settings.gradle.kts

```diff
@@ -80,14 +69,30 @@ gradleEnterprise {
     }
 }
 
+gitRepositories {
+    checkoutsDirectory.set(file(rootProject.projectDir.absolutePath + "/hedera-node/hapi/"))
+    include("hedera-protobufs") {
+        uri.set("https://github.com/hashgraph/hedera-protobufs.git")
+        // choose tag or branch of HAPI you would like to test with
+        // This version needs to match tha HAPI version below in versionCatalogs
+        tag.set("v0.34.0")
```

Comment:
Will changing the value in the IDE automatically update the folder by pulling the given tag?

Reply:
no it is manual to choose the tag, this is the tag for the hedera-protobufs repo so somewhat disconnected from services repo version. They are loosely aligned. It needs to match the version of the hapi library that is imported.